### DISCUSSION TITLE
How can CROCUS be activated in fully coupled WRF and WRF-Hydro/Glacier？

### DISCUSSION THREAD

**Discussion by Lise-Li** [NCAR/wrf_hydro_nwm_public#770] [Q&A]
Hi, I was able to run the fully coupled WRF and WRF-Hydro model, but I was wondering how to add glacier module (Crocus) to the coupled model? How the option of crocus (crocus_opt and act_lev) in offline WRF-Hydro/Glacier's namelist.hrldas document can be set in coupled namelist.input? Also, I want know how the wrfinput should be prepared with the fully coupled WRF and WRF-Hydro/Glacier?
I would appreciate it if you could help me. Looking forward to your reply. Thank you!
![91549ec557b6d0a0147b9b840c62f9a9](https://github.com/user-attachments/assets/881fce76-ec6b-49fb-a145-d998fabaa92e)

**Comment by aubreyd** [comment]
Hi Lise-Li: I don't think Crocus is supported in the coupled WRF/WRF-Hydro mode yet. Tagging @trudeeidhammer to confirm.

**Comment by trudeeidhammer** [comment]
Hi Aubrey

You are correct, Crocus is not supported in the fully coupled WRF. In
WRF-Hydro/Glacier it can be activated by setting
&CROCUS_nlist

  crocus_opt = 1   ! 0 model is off, 1 model is on

  act_lev = 40     ! 20-40 normal range, 1-50 acceptable

in namelist.hrldas

Thanks,
Trude


On Wed, Sep 4, 2024 at 8:05 AM Aubrey Dugger ***@***.***>
wrote:

> Hi Lise-Li: I don't think Crocus is supported in the coupled WRF/WRF-Hydro
> mode yet. Tagging @trudeeidhammer <https://github.com/trudeeidhammer> to
> confirm.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NCAR/wrf_hydro_nwm_public/discussions/770#discussioncomment-10546451>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AD24I5N35QYDRRRKA3OE43DZU4HUNAVCNFSM6AAAAABNUJUSE6VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTANJUGY2DKMI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> com>
>


-- 
*My working day may not be your working day. Please do not feel obliged to
reply to this email outside of your normal working hours.*

She/Her/Hers
Dr. Trude Eidhammer
Project Scientist II
Research Applications Laboratory (RAL)
NSF NCAR
P.O. Box 3000
Boulder, CO 80307-3000, USA

**Comment by Lise-Li** [comment]
Hi, Trude @trudeeidhammer and Aubrey @aubreyd 

Thank you very much for your respond. Apologize for the inconvenience, I have another question to consult with you. Since the Crocus can not be supported in fully coupled WRF, are there any other choices in fully coupled model to explore the 2-way impact of glacier (eg. glacier thickness) to atmosphere and hydrology?
Looking forward to your reply.

Best regards,
Lise
